### PR TITLE
fix windows matrix

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@isuruf and @gillins do you have an idea of what is going on `vc9`?

**Comment by isuruf** [comment]
Change `int` at https://github.com/OSGeo/gdal/blob/release/2.2/gdal/port/cpl_cpu_features.cpp#L205 to `bool`

**Comment by gillins** [comment]
This may be relevant: https://github.com/P-H-C/phc-winner-argon2/issues/44#issuecomment-163347779

But - have we not compiled `libgdal` 2.2.* with `vc9` before?

**Comment by ocefpaf** [comment]
> But - have we not compiled libgdal 2.2.* with vc9 before?

Yes, we did... Many times actually :-(

**Comment by ocefpaf** [comment]
After https://github.com/conda-forge/libgdal-feedstock/pull/55#issuecomment-407582526 we got a little bit further but I'm stuck with a new error:


```
gdalgridavx.cpp
c:\bld\libgdal_1532491241347\work\alg\gdalgridavx.cpp(33) : fatal error C1083: Cannot open include file: 'immintrin.h': No such file or directory
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\amd64\cl.EXE"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\BIN\amd64\nmake.EXE"' : return code '0x2'
Stop.
Traceback (most recent call last):
  File "C:\Miniconda36-x64\Scripts\conda-build-script.py", line 10, in <module>

```

**Comment by isuruf** [comment]
Your issues are because of https://github.com/conda-forge/libgdal-feedstock/blob/2.2.x/recipe/bld.bat#L9-L20
`CONDA_PY` is no longer correct because you don't have `python` as a dependency anymore.
Use `VS_MAJOR` instead.

**Comment by ocefpaf** [comment]
> Your issues are because of https://github.com/conda-forge/libgdal-feedstock/blob/2.2.x/recipe/bld.bat#L9-L20
`CONDA_PY` is no longer correct because you don't have `python` as a dependency anymore.
Use `VS_MAJOR` instead.

Ack! Should've seen that!! Thanks!!!

**Comment by gillins** [comment]
> Use VS_MAJOR instead.

well done @isuruf !

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ source:
     - windowshdf5.patch
     # Use multiple cores on Windows.
     - multiprocessor.patch
+    - win_int_bool.patch  # [win and vc<14]
```
Comment (gillins) [inline recipe/meta.yaml]: Do we still need this?

Comment (ocefpaf) [reply recipe/meta.yaml]: Probably not. Let me try removing it.

**Pull request merged**